config/katello.migrations/200123161606-enable-pulpcore.rb

```diff
@@ -0,0 +1,17 @@
+puppet_class = 'foreman_proxy::plugin::pulp'
```

Comment:
Maybe this needs to be `PUPPET_CLASS` so it's available within the method

Reply:
Once I made the change to the default answers file, that issue disappeared so I think it's OK

Reply:
I think it's now following a different code path where upgrade is not actually executed. That's one way of avoiding build failures, but won't help much in production ;)

Reply:
Good point, thanks :)